modules/default/calendar/calendarfetcher.js

```diff
@@ -117,7 +117,8 @@ const CalendarFetcher = function (url, reloadInterval, excludedEvents, maximumEn
 						endDate = startDate.clone().add(moment.duration(event.duration));
 					} else {
 						if (!isFacebookBirthday) {
-							endDate = startDate;
+							// make copy of start date, separate storage area
+							endDate = moment(startDate.format("x"), "x");
```

Comment:
this not change anything, same plus one day for recuring events

Reply:
can u post a vevent from  your calendar?

and u must be on the develop branch

Reply:
also need this other fix.. fulldayevent checker

Reply:
I am on develp branch, and not working any fix
this is calendar with weekly recurrent events

BEGIN:VCALENDAR
PRODID:-//Google Inc//Google Calendar 70.9054//EN
VERSION:2.0
CALSCALE:GREGORIAN
METHOD:PUBLISH
X-WR-CALNAME:Filme si seriale
X-WR-TIMEZONE:Europe/Bucharest
BEGIN:VEVENT
DTSTART;VALUE=DATE:20201017
DTEND;VALUE=DATE:20201018
RRULE:FREQ=WEEKLY;BYDAY=SA
DTSTAMP:20201026T212524Z
UID:4ulvpcckkq5cclafbam43cmgok@google.com
CREATED:20201025T194923Z
DESCRIPTION:
LAST-MODIFIED:20201025T194923Z
LOCATION:
SEQUENCE:0
STATUS:CONFIRMED
SUMMARY:Fear TWD
TRANSP:TRANSPARENT
END:VEVENT
BEGIN:VEVENT
DTSTART;VALUE=DATE:20201030
DTEND;VALUE=DATE:20201031
RRULE:FREQ=WEEKLY;BYDAY=FR
DTSTAMP:20201026T212524Z
UID:7uaf5c5c8bt9cuaarcqplqpdt5@google.com
CREATED:20201025T194842Z
DESCRIPTION:
LAST-MODIFIED:20201025T194842Z
LOCATION:
SEQUENCE:0
STATUS:CONFIRMED
SUMMARY:Mandalorian
TRANSP:TRANSPARENT
END:VEVENT
BEGIN:VEVENT
DTSTART;VALUE=DATE:20201015
DTEND;VALUE=DATE:20201016
RRULE:FREQ=WEEKLY;BYDAY=TH
DTSTAMP:20201026T212524Z
UID:1ia5llv69vra7ipiusv750idq2@google.com
CREATED:20201025T194810Z
DESCRIPTION:
LAST-MODIFIED:20201025T194811Z
LOCATION:
SEQUENCE:0
STATUS:CONFIRMED
SUMMARY:Star Trek
TRANSP:TRANSPARENT
END:VEVENT
BEGIN:VEVENT
DTSTART;VALUE=DATE:20210319
DTEND;VALUE=DATE:20210320
DTSTAMP:20201026T212524Z
UID:2pq91nfe4tbd8gn72u13viurgh@google.com
CREATED:20201024T143644Z
DESCRIPTION:
LAST-MODIFIED:20201024T143645Z
LOCATION:
SEQUENCE:0
STATUS:CONFIRMED
SUMMARY:Morbius
TRANSP:TRANSPARENT
END:VEVENT
BEGIN:VEVENT
DTSTART;VALUE=DATE:20201106
DTEND;VALUE=DATE:20201107
DTSTAMP:20201026T212524Z
UID:33nrfj1v0gp24shfvsek87hilo@google.com
CREATED:20201024T143605Z
DESCRIPTION:
LAST-MODIFIED:20201024T143605Z
LOCATION:
SEQUENCE:0
STATUS:CONFIRMED
SUMMARY:Black Widow
TRANSP:TRANSPARENT
END:VEVENT
BEGIN:VEVENT
DTSTART;VALUE=DATE:20201218
DTEND;VALUE=DATE:20201219
DTSTAMP:20201026T212524Z
UID:7ovdici1g552lud414vtouv5n1@google.com
CREATED:20201024T143539Z
DESCRIPTION:
LAST-MODIFIED:20201024T143539Z
LOCATION:
SEQUENCE:0
STATUS:CONFIRMED
SUMMARY:Dune
TRANSP:TRANSPARENT
END:VEVENT
BEGIN:VEVENT
DTSTART;VALUE=DATE:20210117
DTEND;VALUE=DATE:20210118
DTSTAMP:20201026T212524Z
UID:76c85tvoftuvoge1591ntpvt9v@google.com
CREATED:20201024T143505Z
DESCRIPTION:
LAST-MODIFIED:20201024T143505Z
LOCATION:
SEQUENCE:0
STATUS:CONFIRMED
SUMMARY:Resident Alien SciFi
TRANSP:TRANSPARENT
END:VEVENT
BEGIN:VEVENT
DTSTART;VALUE=DATE:20201216
DTEND;VALUE=DATE:20201217
DTSTAMP:20201026T212524Z
UID:6g2uthpr4ebr9t841hk73cu4r3@google.com
CREATED:20201023T203651Z
DESCRIPTION:
LAST-MODIFIED:20201023T203651Z
LOCATION:
SEQUENCE:0
STATUS:CONFIRMED
SUMMARY:The Expanse 5
TRANSP:TRANSPARENT
END:VEVENT
BEGIN:VEVENT
DTSTART;VALUE=DATE:20201122
DTEND;VALUE=DATE:20201123
DTSTAMP:20201026T212524Z
UID:24q2mlu5r3pkqrofmdhac9bbjd@google.com
CREATED:20201017T202438Z
DESCRIPTION:
LAST-MODIFIED:20201017T202438Z
LOCATION:
SEQUENCE:0
STATUS:CONFIRMED
SUMMARY:JIU JITSU
TRANSP:TRANSPARENT
END:VEVENT
BEGIN:VEVENT
DTSTART;VALUE=DATE:20201024
DTEND;VALUE=DATE:20201025
DTSTAMP:20201026T212524Z
UID:07d900mijh8tmt0c7koht044fv@google.com
CREATED:20200626T171925Z
DESCRIPTION:
LAST-MODIFIED:20201017T062112Z
LOCATION:
SEQUENCE:16
STATUS:CONFIRMED
SUMMARY:greenland
TRANSP:TRANSPARENT
BEGIN:VALARM
ACTION:NONE
TRIGGER;VALUE=DATE-TIME:19760401T005545Z
END:VALARM
END:VEVENT
BEGIN:VEVENT
DTSTART;VALUE=DATE:20201120
DTEND;VALUE=DATE:20201121
DTSTAMP:20201026T212524Z
UID:0okvrt9m0cft1kfqmugao54qq5@google.com
CREATED:20201009T162954Z
DESCRIPTION:
LAST-MODIFIED:20201009T162954Z
LOCATION:
SEQUENCE:0
STATUS:CONFIRMED
SUMMARY:Delicate Sound of Thunder
TRANSP:TRANSPARENT
END:VEVENT
BEGIN:VEVENT
DTSTART;VALUE=DATE:20201209
DTEND;VALUE=DATE:20201210
DTSTAMP:20201026T212524Z
UID:25oqfu8soeus7osviadqrvr9g7@google.com
CREATED:20200905T182614Z
DESCRIPTION:
LAST-MODIFIED:20200909T105950Z
LOCATION:
SEQUENCE:1
STATUS:CONFIRMED
SUMMARY:Monsters of men
TRANSP:TRANSPARENT
END:VEVENT
END:VCALENDAR

Reply:
Ok, this is what I see , I am 1 fix up from develop and this fix.. as we are waiting for node-ical to release fix..
```
  | Star Trek | In 3 days
  | Mandalorian | In 4 days
  | Fear TWD | In 5 days
  | Star Trek | In 10 days
  | Mandalorian | In 11 days
  | Black Widow | In 11 days
  | Fear TWD | In 12 days
  | Star Trek | In 17 days
  | Mandalorian | In 18 days
  | Fear TWD | In 19 days
```

I can send updated files to replace existing to test.. email me at this same id at gmail.com to get the files

Reply:
sent